data/KNC/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Kyber Network Crystal v2",
+  "symbol": "KNC",
+  "decimals": 18,
+  "tokens": {
+    "ethereum": {
+      "address": "0xdeFA4e8a7bcBA345F687a2f1456F5Edd9CE97202"
+    },
+    "optimism": {
+      "address": "0xa00E3A3511aAC35cA78530c85007AFCd31753819"
+    },
+    "kovan": {
```

Comment:
You will need to include an OptimismKovan address or remove this one for now

Reply:
PR updated, removed KNC kovan address